changelogs/unreleased/gh-9643-credentials-from-app-roles.md

```diff
@@ -0,0 +1,4 @@
+## feature/config
+
+* A non-existent role can now be assigned in the `credential` section of
+  the configuration.
```

Comment:
While it is actually a small enhancement for the credentials applier (so, we can call it a feature), it is part of a bugfix in a more wide context of tarantool itself. We need this change to fix `box.schema.user.grant()` clash with the credentials applier (#9643). We should backport this change to 3.0. So, I vote for the labeling it as a bugfix.

Reply:
True, fixed. Also, added some text to commit-message.